src/views/Routes.tsx

```diff
@@ -0,0 +1,384 @@
+        dispatch(routeActions.setSetupNewRouteVisible(true));
+        dispatch(routeActions.setRoute({
+            network: '',
+            network_id: '',
+            description: '',
+            peer: '',
+            masquerade: false,
+            metric: 9999,
+            enabled: false
+        } as Route))
+    }
+
+    const onClickViewRoute = () => {
+        dispatch(routeActions.setSetupNewRouteVisible(true));
+        dispatch(routeActions.setRoute({
+            id: routeToAction?.id || null,
+            network: routeToAction?.network,
+            network_id: routeToAction?.network_id,
+            description: routeToAction?.description,
+            peer: routeToAction?.peer,
+            metric: routeToAction?.metric,
+            masquerade: routeToAction?.masquerade,
+            enabled: routeToAction?.enabled
+        } as Route))
+    }
+
+    const setRouteAndView = (route: RouteDataTable) => {
+        dispatch(routeActions.setSetupNewRouteVisible(true));
+        dispatch(routeActions.setRoute({
+            id: route.id || null,
+            network: route.network,
+            network_id: route.network_id,
+            description: route.description,
+            peer: route.peer,
+            metric: route.metric,
+            masquerade: route.masquerade,
+            enabled: route.enabled
+        } as Route))
+    }
+
+    const showConfirmEnableMasquerade = (record: RouteDataTable, checked: boolean) => {
+        let label = record.network_id ? record.network_id : record.network
+        let tittle = "Enable Masquerade for \"" + label + "\"?"
+        let content = "Enabling this option hide all traffic coming from other NetBird peers through the routing peer address."
+
+        if (!checked) {
+            tittle = "Disable Masquerade for \"" + label + "\"?"
+            content = "Disabling this option stops hiding all traffic coming from other NetBird peers through the routing peer address."
+        }
+
+        confirm({
+            icon: <ExclamationCircleOutlined />,
+            title: tittle,
+            width: 600,
+            content: content,
+            okType: 'danger',
+            onOk() {
+                handleSwitchMasquerade(record, checked)
+            },
+            onCancel() {
+            },
+        });
+    }
+
+    function handleSwitchMasquerade(record: RouteDataTable, checked: boolean) {
+        const route = {
+            ...record,
+            masquerade: checked,
+        } as Route
+        dispatch(routeActions.saveRoute.request({getAccessTokenSilently:accessToken, payload: route}));
+    }
+
+    return(
+        <>
+            <Container className="container-main">
+                <Row>
+                    <Col span={24}>
+                        <Title level={4}>Network Routes</Title>
+                        <Paragraph>Network routes allow you to create routes to network ranges and assign NetBird peers as routers for these ranges.</Paragraph>
```

Comment:
I think the Network Route tab description should be more specific to what problem it solves. E.g. instead of

Network routes allow you to create routes to network ranges and assign NetBird peers as routers for these ranges.

Point out that users will be able to access networks without installing NetBird on every resource.

Reply:
suggestion: 

Network routes allow you to create routes and access other networks without installing NetBird on every resource.

Reply:
Sounds good maybe replace "and" with "to"?